backend/src/Designer/Controllers/ResourceAdminController.cs

```diff
@@ -152,6 +160,71 @@ public ActionResult<ServiceResource> AddResource(string org, [FromBody] ServiceR
             return _repository.AddServiceResource(org, resource);
         }
 
+
+
+        [HttpGet]
+        [Route("designer/api/{org}/resources/sectors")]
+        public async Task<ActionResult<List<DataTheme>>> GetSectors()
+        {
+            string cacheKey = "sectors";
+            if (!_memoryCache.TryGetValue(cacheKey, out List<DataTheme> sectors))
+            {
+                DataThemesContainer dataThemesContainer = await _resourceRegistryOptions.GetSectors();
+
+                var cacheEntryOptions = new MemoryCacheEntryOptions()
+               .SetPriority(CacheItemPriority.High)
+               .SetAbsoluteExpiration(new TimeSpan(0, 3600, 0));
```

Comment:
Should we add this cache timer to a configuration?

Reply:
Added